### DISCUSSION TITLE
How does this compare with Radius?

### DISCUSSION THREAD

**Discussion by onionhammer** [microsoft/aspire#811] [Q&A] [answered]
https://github.com/radius-project/radius

Seems like they're mutually exclusive, both recently announced Microsoft ways of creating distributed/multi-project applications, leveraging bicep/etc,

(Also, there's tye)

**Comment by KennethHoff** [comment]
There's also [Orleans](https://github.com/dotnet/orleans). How does that fit into this?

**Reply by davidfowl** [reply]
Orleans will work *with* aspire! Track [this example](https://github.com/dotnet/aspire-samples/issues/20) for updates to that cc @ReubenBond @benjaminpetit. Orleans runs within an orchestrator and the aspire app host just happens to be one.

**Reply by ReubenBond** [reply]
Orleans support has now shipped: https://learn.microsoft.com/en-us/dotnet/aspire/whats-new/preview-3#add-support-for-orleans-in-net-aspire

**Comment by davidfowl** [comment]
@onionhammer 

> Seems like they're mutually exclusive, both recently announced Microsoft ways of creating distributed/multi-project applications, leveraging bicep/etc,

I don't see why. Radius is very much interested in the developer to devops hand off but not the inner loop. We actually think it's a pretty nice end to end for:

aspire manifest -> radius bicep -> wherever radius recipes let you deploy

Ryan (Architect of Radius) and I have been discussion the PoC we're going to build as the first step here.

**Reply by dosper7** [reply]
Can't wait to see that 👍

**Reply by mkane91301** [reply]
Came here to ask for that. Glad it's happening.

**Reply by thangchung** [reply]
Yeah. Looking forward to seeing Aspire and Radius fit together 👀

**Reply by sclarke81** [reply]
Same here. I'd really love to see an example that uses Dapr, Aspire and Redius. It seems like this combination could create a fantastic development and deployment experience.

**Reply by YohanSciubukgian** [reply]
@davidfowl Do you have any update on the POC and integration phases planned with Radius project (and timeline if any) ?

**Reply by davidfowl** [reply]
Nothing concrete yet but there have been recent discussions.

**Comment by akhanalcs** [comment]
@davidfowl What's going to happen to Tye?

**Reply by davidfowl** [reply]
The repository will be archived and no work will happen on it.

**Comment by davidfowl** [comment, marked as answer]
Radis team just merged their first PR https://github.com/microsoft/aspire/pull/18696
